python/ray/data/tests/test_tfrecords.py

```diff
@@ -338,9 +338,13 @@ def _str2bytes(d):
     assert ds_expected.take() == ds_actual.take()
 
 
-@pytest.mark.parametrize("with_tf_schema", (True, False))
+@pytest.mark.parametrize(
```

Comment:
for tests, we also need to install tfx-bsl with `--no-dependencies`, where should this be added?